src/stan-users-guide/regression.Rmd

```diff
@@ -1411,7 +1411,7 @@ beta ~ multi_normal(...);
 which requires `beta` to be an array of vectors.  Both
 vectorizations are important, so the best solution is to just use the
 loop above, because `rows_dot_product` cannot do much
-optimization in and of itself because there are no shared computations.
+optimization in and of itself since there are no shared computations.
```

Comment:
[comment & optional change]
I was trained to use "because" instead of "since" to avoid ambiguity with the temporal ordering.  I should point out that I was a computational linguistics Ph.D. student in the U.K. whose thesis was on tense and aspect, so I might've been overly sensitive at the time.  I haven't been able to use "since" as a syonym for "because" since!

Anyway, no big deal, just an explanation of why the docs say "because" everywhere.

Reply:
So I can press resolve conversation? Or should you, since you initiated it? Sorry, never used this GitHub interface before :)

Reply:
Sure.  These are all very light suggestions and I'm just going to leave the fixes up to you.  So feel free to resolve or leave unresolved or whatever's easiest.  They're enot part of the formal process.

Reply:
Changed